### PR TITLE
feat(Metrics): Add distributor Metrics.event API

### PR DESCRIPTION
**Pull request by jlacivita**

### CODE REVIEW

File: src/openrpc/metrics.json [hunk 1]
```diff
@@ -1039,6 +1097,56 @@
...
+        ]
+      },
+      "EventObject": {
+        "title": "EventObject",
+        "type": "object",
+        "maxProperties": 256,
+        "additionalProperties": {
+          "anyOf": [
+            {
+              "$ref": "#/components/schemas/EventObjectPrimatives"
```
Comment (kpears201) [inline src/openrpc/metrics.json:1129]: Primitives misspelled here

Comment (jlacivita) [reply src/openrpc/metrics.json]: fixed

**Review by kpears201 — APPROVED** [review]

**Pull request merged**